docs/audit/audit-persistence.md

```
@@ -0,0 +1,520 @@
+That part is correct, but the readability cost is real.
+**Why it matters:** Style / maintainability with a latent correctness trap.
+**Suggested fix:** Extract a helper:
+```kotlin
+fun bestPlacementAcross(modes: List<PlayerStats>): Int =
+    modes.mapNotNull { it.bestPlacement.takeIf { p -> p > 0 } }.minOrNull() ?: 0
+```
+**Confidence:** low (style-leaning)
+
+---
+
+### [SEV: med] persistence/DatabaseService.kt:99-139 — `migrateSqliteLegacyArenasTable` has no `try/finally` around `PRAGMA foreign_keys`
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)